packages/app/components/Questions/Queue/TaskMarkingSelector.tsx

```diff
@@ -0,0 +1,158 @@
+import { ReactElement, useState } from 'react'
+import { Text } from '../Shared/SharedComponents'
+import { ConfigTasks } from '@koh/common'
+
+/*
+  This component is used when the TA is helping a student (used on the student queue card). 
+  Allows the TA to choose what tasks (of the one's the student is asking to be marked) are good and should be marked 
+*/
+
+interface CheckableMarkingTaskProps {
+  taskId: string
+  taskName: string
+  taskColor: string
+  onChange: (taskId: string, checked: boolean) => void
+  checked: boolean
+  decorative: boolean
+}
+
+function CheckableMarkingTask({
+  taskId,
+  taskName,
+  taskColor,
+  onChange,
+  checked,
+  decorative,
+}: CheckableMarkingTaskProps): React.ReactElement {
+  function getBrightness(color: string): number {
```

Comment:
Do we have this method already?

Reply:
yeah i mean technically it's within QuestionType (which is what this file was originally based off of) and technically they can be refactored to both use the same function in a shared file but meh i don't think it's worth changing at this point. Maybe if we use getBrightness in a 3rd area, i'll refactor it